run_dir/design/projects.html

```diff
@@ -484,6 +484,47 @@ <h4>Delete Preset</h4>
   </div>
 </div>
 
+<!-- <div id="saveFilterModal" class="modal fade">
```

Comment:
Should be removed if this is not used. We can add it back later when the functionality is needed

Reply:
Ooops!